content/en/docs/topics/registries.md

```diff
@@ -56,6 +56,7 @@ There are several hosted container registries with OCI support that you can use
 - [Azure Container Registry](https://docs.microsoft.com/azure/container-registry/container-registry-helm-repos#push-chart-to-registry-as-oci-artifact)
 - [Docker Hub](https://docs.docker.com/docker-hub/oci-artifacts/)
 - [Google Artifact Registry](https://cloud.google.com/artifact-registry/docs/helm/manage-charts)
+- [Harbor](https://goharbor.io/)
```

Comment:
@OrlinVasilev could you use link that about OCI? like https://docs.docker.com/docker-hub/oci-artifacts/.

Reply:
hi @yxxhero currently we don't have a specific page for OCI helm charts so that's the closest, but it's a good addon to our documentation so I'll work on such document and will update the PR again :) Thank you!

Reply:
@OrlinVasilev  is https://goharbor.io/docs/main/administration/user-defined-oci-artifact/ latest? thanks.

Reply:
not latest but always with the current GA version!